extensions/amp-script/0.1/amp-script.js

```diff
@@ -158,6 +156,11 @@ export class AmpScript extends AMP.BaseElement {
    * @private
    */
   mutationPump_(flush, phase) {
+    if (phase == PHASE_HYDRATING) {
+      this.vsync_.mutate(() => {
```

Comment:
Nit: No need for curlies here. 

`this.vsync_.mutate(() => this.element.classList.add('i-amphtml-hydrated'))`